core/charm/computedseries.go

```diff
@@ -36,5 +36,10 @@ func ComputedSeries(c charm.CharmMeta) ([]string, error) {
 			seriesSlice = append(seriesSlice, series)
```

Comment:
Are we using the slice instead of the set (with a .Values/SortedValues() in the end) here to maintain the order as seen in the bases?

Reply:
Yes

Reply:
Doh... didn't read the comment either!